apps/quartz-app/src/components/Main.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
```

Comment:
should this be `main.tsx` not `Main.tsx`

Reply:
Nah, React components are always capitalised by convention